website/static/js/comment.js

```diff
@@ -302,7 +302,7 @@ CommentModel.prototype.submitEdit = function(data, event) {
         self.content(response.content);
         self.dateModified(response.dateModified);
         self.editing(false);
-        self.modified(true);
+        self.modified = true;
```

Comment:
`self.modified` is only used by `editVerb`, which appears to be unused. Perhaps we just remove `modified` and `editVerb`. @Ghalko can you see if this is do-able?